src/commands/aksCreateCluster/aksCreateCluster.ts

```diff
@@ -21,10 +26,23 @@ export default async function aksCreateCluster(_context: IActionContext, target:
         return;
     }
 
-    const subscriptionNode = getAksClusterSubscriptionNode(target, cloudExplorer);
-    if (failed(subscriptionNode)) {
-        vscode.window.showErrorMessage(subscriptionNode.error);
-        return;
+    if (typeof target !== "string") {
+        subscriptionNode = getAksClusterSubscriptionNode(target, cloudExplorer);
+        if (failed(subscriptionNode)) {
+            vscode.window.showErrorMessage(subscriptionNode.error);
+            return;
+        } else {
+            subscriptionId = subscriptionNode.result?.subscriptionId;
```

Comment:
💡 any case where this is going to be `nullable` type? `subscriptionNode.result?`  or its guranteed result if so can we do `!` or handle `nullable` types in error handling.